### ISSUE TITLE
"NaN" results for W values

### ISSUE THREAD

**Issue by thermengineer** [CoolProp/CoolProp#1822]
Dear Colleagues,
We got several "NaN" results on version 6.2.1. You can see them below. What could be the reason? Thank you for your help beforehand.

1-CoolProp.HAPropsSI(“W”, “T”, 289.51802102604, “P”, 101325, “H”, 44863.36107700791)= 0.01122519678493846
2-CoolProp.HAPropsSI(“W”, “T”, 290.67055539281, “P”, 101325, “H”, 47727.304155665959)= 0.011888898975338824
3-CoolProp.HAPropsSI(“W”, “T”, 292.35196500039, “P”, 101325, “H”, 52183.318550432617)= 0.012964759480085689
4-CoolProp.HAPropsSI(“W”, “T”, 291.30665075216, “P”, 101325, “H”, 49372.44716514201)= 0.012280186436999355
5-CoolProp.HAPropsSI(“W”, “T”, 291.75699599256, “P”, 101325, “H”, 50566.581308015782)=NaN
6-CoolProp.HAPropsSI(“W”, “T”, 292.03241812858, “P”, 101325, “H”, 51309.352071811714)= 0.012749800862113794
7- CoolProp.HAPropsSI(“W”, “T”, 291.88977606563, “P”, 101325, “H”, 50923.468854566643) =NaN
8- CoolProp.HAPropsSI(“W”, “T”, 291.95979778379, “P”, 101325, “H”, 51112.571586160921)= 0.012701660315375713
9- CoolProp.HAPropsSI(“W”, “T”, 291.99577429328, “P”, 101325, “H”, 51209.973476403233)= 0.012725476808875619
10- CoolProp.HAPropsSI(“W”, “T”, 291.97770375168, “P”, 101325, “H”, 51161.029109086929) =NaN
11- CoolProp.HAPropsSI(“W”, “T”, 291.98671831104, “P”, 101325, “H”, 51185.439984518176) =NaN
12- CoolProp.HAPropsSI(“W”, “T”, 291.99124110641, “P”, 101325, “H”, 51197.691346337415)= 0.012722472314019533
13- CoolProp.HAPropsSI(“W”, “T”, 291.99350639895, “P”, 101325, “H”, 51203.82855817581) =NaN
14- CoolProp.HAPropsSI(“W”, “T”, 291.99237342771, “P”, 101325, “H”, 51200.7589898548)= 0.012723222695963834
15- CoolProp.HAPropsSI(“W”, “T”, 291.99293983197, “P”, 101325, “H”, 51202.293533302836) =NaN
16- CoolProp.HAPropsSI(“W”, “T”, 291.99322309506, “P”, 101325, “H”, 51203.060985547192) =NaN
17- CoolProp.HAPropsSI(“W”, “T”, 291.99308145863, “P”, 101325, “H”, 51202.677244378734) =NaN
18- CoolProp.HAPropsSI(“W”, “T”, 291.99301064408, “P”, 101325, “H”, 51202.485385079432) =NaN

**Comment by jowr** [comment]
If errors occur, it is always a good start to check the output of the error string: 
```
import CoolProp.CoolProp as CP
print(CP.get_global_param_string("warnstring"))
print(CP.get_global_param_string("errstring"))
```

**Comment by thermengineer** [comment]
Thank you for your answer.  (We use c#, asp.net mvc) What we got is mentioned below. No value returned. Thank you beforehand for your response.
        {
            double ret = CoolPropPINVOKE.HAPropsSI(OutputName, Input1Name, Input1, Input2Name, Input2, Input3Name, Input3);
            var val1=CoolPropPINVOKE.get_global_param_string("warnstring");
            var val2 = CoolPropPINVOKE.get_global_param_string("errstring");

            if (CoolPropPINVOKE.SWIGPendingException.Pending) throw CoolPropPINVOKE.SWIGPendingException.Retrieve();
            return ret;
        }
        catch (System.Exception ex)
        {
            var a = ex.Message;
            throw;
        }
Returns
Val1=””
Val2=””

**Comment by ibell** [comment]
We have a fix underway in #1823 ; just needs some brief testing, and this should fix your problem.  Will be released in version 6.3

**Comment by thermengineer** [comment]
Dear Ian, thank you for your answer. We wait 6.3 version. Best regards
